Assets/MixedRealityToolkit/Inspectors/Profiles/MixedRealityInputSystemProfileInspector.cs

```diff
@@ -273,11 +273,7 @@ private void RenderList(SerializedProperty list)
                                 EditorGUILayout.PropertyField(runtimePlatform, RuntimePlatformContent);
                                 changed |= EditorGUI.EndChangeCheck();
 
-                                System.Type serviceType = null;
-                                if (configurationProfile.objectReferenceValue != null)
-                                {
-                                    serviceType = (target as MixedRealityInputSystemProfile).DataProviderConfigurations[i].ComponentType;
-                                }
+                                System.Type serviceType = (target as MixedRealityInputSystemProfile).DataProviderConfigurations[i].ComponentType;
 
                                 changed |= RenderProfile(configurationProfile, null, true, false, serviceType);
                             }
```

Comment:
You may need to add the typeof() check as in spatial

Reply:
No actually, the profile type under DataProviderConfigurations is actually BaseMixedRealityProfile
![image](https://user-images.githubusercontent.com/25975362/66958478-18a9a680-f01d-11e9-9841-ee24c34256ce.png)

While the ObserverConfigurations in SA, explicitly use BaseSpatialAwarenessObserverProfile

![image](https://user-images.githubusercontent.com/25975362/66958523-2d863a00-f01d-11e9-9629-f6f2960a31cc.png)